code/__HELPERS/atoms.dm

```diff
@@ -350,3 +350,17 @@ B --><-- A
 		return get_step(ref, base_dir)
 
 */
+
+/// returns a mob that possesses this atom(usually item)
+/atom/proc/get_mob_owner(returns_final=FALSE)
+	var/atom/upper = src
```

Comment:
this would be better named `get_containing_mob()` and `returns_final` as `first_mob`

Reply:
`proc/get_loc_mob(as_first_mob=TRUE)` will be cleaner. now changed.